etc/systemd/aquilon-broker.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Quattor Aquilon broker
+After=network.target
+
+[Service]
+EnvironmentFile=/etc/sysconfig/aqd
```

Comment:
is this file provided by a package (like bidst_rpm)? if not, make it optional by precedingthe filename with a `-`

Reply:
Yes, it is part of the Aquilon distribution and the installation documentation explains how to customize it. It is **required** to exist to be able to run the broker.